types/graphql/type/definition.d.ts

```diff
@@ -36,35 +36,35 @@ export function isType(type: any): type is GraphQLType;
 
 export function assertType(type: any): GraphQLType;
 
-export function isScalarType(type: GraphQLType): type is GraphQLScalarType;
+export function isScalarType(type: any): type is GraphQLScalarType;
 
 export function assertScalarType(type: GraphQLType): GraphQLScalarType;
```

Comment:
@benwaffle To keep definitions consistent can you please also change `assert*Type` functions?

Reply:
updated